docs/release_notes/v1.0.0-beta2.adoc

```diff
@@ -0,0 +1,171 @@
+https://download.elastic.co/downloads/ecctl/1.0.0-beta2/ecctl_1.0.0-beta2_linux_64-bit.deb[ecctl_1.0.0-beta2_linux_64-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.0.0-beta2/ecctl_1.0.0-beta2_linux_64-bit.rpm[ecctl_1.0.0-beta2_linux_64-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.0.0-beta2/ecctl_1.0.0-beta2_linux_amd64.tar.gz[ecctl_1.0.0-beta2_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.0.0-beta2-whats-new"]
+==== What's new
+
+* *AppSearch instances included in default vacate operation* .App Search instances will now be moved with the default `platform allocator vacate` command. Although App Search workloads were supported in the previous release, support for vacating said instances was missed.
+
+* *New Commands*. We added a number of new commands to implement API functionality:
++
+--
+* Deployment appsearch commands:
+
+`ecctl deployment appsearch create`:: Creates an AppSearch instance.
+`ecctl deployment appsearch delete`:: Deletes a previously shut down AppSearch deployment resource.
+`ecctl deployment appsearch show`:: Shows the specified AppSearch deployment.
+`ecctl deployment appsearch shutdown`:: Shuts down an AppSearch deployment.
+`ecctl deployment appsearch upgrade`:: Upgrades an AppSearch deployment to the Elasticsearch deployment version.
+
+* We also added a few subcommands to the platform runner command:
++
+
+`ecctl platform runner list`::  Lists the existing platform runners
+`ecctl platform runner resync`:: Resynchronizes the search index and cache for the selected runner or all
+`ecctl platform runner search`:: Performs advanced runner searching
+`ecctl platform runner show`:: Shows information about the specified runner
+--
+
+* *--kind flag autocompletion*`. Commands which have a `--kind` flag, now have shell autocompletion for the allowed kinds for the command.
+
+* *Improved documentation*. We now include the full {p} command reference with our official documentation. We also added auto-completion instructions to the docs.
+
+* *Added an `--all` flag for all `resync` commands*. Allowing to trigger a resync not only for a specific id but for all of the indexed records.
+
+* *Changes the default polling period for the `platform allocator vacate` operation*. Defaults to 10s instead of 1s. This change should reduce the computational impact that the `allocator vacate` operation has in both the machine running the command and the target environment where the vacate is being run.
+
+* *Adds --max-poll-retries and --poll-frequency flags to `platform allocator vacate`*. Allowing users to configure these settings if needed.
+
+* *Adds a `--plan-history` flag to `deployment show`*. To be able to obtain the deployment plan history from a specific deployment.
+
+[float]
+[id="{p}-release-notes-v1.0.0-beta2-bug-fixes"]
+==== Bug fixes
+
+* `ecctl init` returned a cryptic error when the API Key could not be verified. It now returns a human-readable error which the user can act on.
```

Comment:
```suggestion
* `ecctl init` returned a cryptic error when the API Key could not be verified. It now returns a human-readable error which you can act on.
```

(We should address audiences directly, it's part of our human and friendly tone.)